docs/site/Importing-LB3-models.md

```diff
@@ -0,0 +1,28 @@
+---
+lang: en
+title: 'Importing models from LoopBack 3 projects'
+keywords: LoopBack 4.0, Migration
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Importing-LB3-models.html
+---
+
+## Synopsis
+
+To simplify migration from LoopBack 3, LoopBack 4 provides a CLI tool to import
+LoopBack 3 models into your LoopBack 4 project.
+
+## Overview
+
+A single model can be imported by running `lb4 import-model` command.
```

Comment:
I wonder if the command name can be better as `lb4 import-lb3-models` or `lb4 import-v3-models`?

Reply:
Good point. Let's go with `lb4 import-lb3-model`.